OpenCombine.podspec

```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |spec|
+  spec.name         = "OpenCombine"
+  spec.version      = "0.0.1"
+  spec.summary      = "Open source implementation of Apple's Combine framework for processing values over time."
+
+  spec.description  = <<-DESC
+  Open source implementation of Apple's Combine framework for processing values over time.
+  DESC
+
+  spec.homepage     = "https://github.com/broadwaylamb/OpenCombine/"
+
+  spec.license      = "Apache 2.0"
+
+  spec.author       = "Apple"
+  spec.source       = { :git => "https://github.com/broadwaylamb/OpenCombine", :tag => "#{spec.version}" }
+
+  spec.swift_version = '5.0'
+
+  spec.osx.deployment_target     = '10.9'
+  spec.ios.deployment_target     = '8.0'
```

Comment:
We also support tvOS and watchOS, do we need to add them here as well?

Reply:
@broadwaylamb yes, what's the minimum deployment target for those platforms?

Reply:
I think it's:

```ruby
spec.osx.deployment_target     = '10.10'
spec.ios.deployment_target     = '8.0'
spec.watchos.deployment_target = '2.0'
spec.tvos.deployment_target    = '9.0'
```

Reply:
Cool, thanks. :) 

I opened a PR on GottaGoFast to support Cocoapods, so that we can use it in the test_spec. 
https://github.com/broadwaylamb/GottaGoFast/pull/1

Will use the same deployment targets there.

Reply:
Oh. I've dropped this dependency recently, so we won't need that :)

Reply:
Oh nice, I'll create a fresh fork so that I'm working with the latest state. :D

Reply:
It's actually:

```
spec.osx.deployment_target     = '10.12'
spec.ios.deployment_target     = '10.0'
spec.watchos.deployment_target = '3.0'
spec.tvos.deployment_target    = '10.0'
```

because of 'os_unfair_lock' not being available before these versions.

Reply:
There is a check here, so it's OK.
https://github.com/broadwaylamb/OpenCombine/blob/c8058edc5f2bee63d9abf2e18c62d5a876da35e2/Sources/COpenCombineHelpers/COpenCombineHelpers.cpp#L181-L195

Reply:
Also, [we have a CI job running on iOS 9.3](https://app.circleci.com/jobs/github/broadwaylamb/OpenCombine/330).

Reply:
@broadwaylamb Version 0.5.0 still depends on `GottaGoFast`. Could you bump the version number on the master branch so that I don't need to integrate that dependency in the test_spec?

Reply:
Another alternative would be to not expose the tests in a test_spec for now, which would be fine I guess. We can tackle that later.

Reply:
This approach seems good to me.